src/constants/pinning.js

```diff
@@ -23,6 +23,12 @@ const pinningServiceTemplates = [
     icon: 'https://dweb.link/ipfs/QmVYXV4urQNDzZpddW4zZ9PGvcAbF38BnKWSgch3aNeViW?filename=pinata.svg',
     apiEndpoint: 'https://api.pinata.cloud/psa',
     visitServiceUrl: 'https://pinata.cloud/documentation#PinningServicesAPI'
+  },
+  {
+    name: 'Filebase',
+    icon: 'https://dweb.link/ipfs/QmWBaeu6y1zEcKbsEqCuhuDHPL3W8pZouCPdafMCRCSUWk?filename=filebase.png',
+    apiEndpoint: 'https://api.filebase.io/v1/ipfs',
+    visitServiceUrl: 'https://docs.filebase.com/api-documentation/ipfs-pinning-service-api'
```

Comment:
All values look good here and filebase is passing 100% on pinning service compliance checker. Will check the icon in webui before merging.

Reply:
![image](https://user-images.githubusercontent.com/1173416/189778533-61ac2aef-3d33-40e7-98f8-b0ef7e6118fb.png)

looks good.